catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/FilterToTextDelegate.java

```diff
@@ -363,6 +381,10 @@ public String propertyIsLessThanOrEqualTo(String propertyName, Object literal) {
         return propertyName + "<=" + literal + "o";
     }
 
+    public String propertyIsDivisibleBy(String propertyName, long divisor) {
+        return propertyName + "%" + divisor + "i=0";
```

Comment:
should be an 'L' following the naming scheme above which uses a letter to describe the type of literal